src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Input/Keyboard/Keyboard_Events.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Windows_UI_Xaml_Input.Keyboard
+{
+	[Sample("Keyboard")]
+	public sealed partial class Keyboard_Events : Page
+	{
+		public Keyboard_Events()
+		{
+			this.InitializeComponent();
+
+			SetupEvent(_root);
+			SetupEvent(_btt1);
+			SetupEvent(_btt2);
+		}
+
+		private void SetupEvent(FrameworkElement elt)
+		{
+			elt.KeyDown += (snd, e) =>
+			{
+				Console.WriteLine($"{elt.Name} - [KEYDOWN] {e.Key}");
+				global::System.Diagnostics.Debug.WriteLine($"{elt.Name} - [KEYDOWN] {e.Key}");
+				_output.Text += $"{elt.Name} - [KEYDOWN] {e.Key}\r\n";
+			};
+			elt.KeyUp += (snd, e) =>
+			{
+				Console.WriteLine($"{elt.Name} - [KEYUP] {e.Key}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5545012)